docs/drivers/auth-keystone.md

```diff
@@ -43,4 +43,4 @@ All policy rules (except for `keppel:admin`) can use the object attribute `%(tar
 See also: [List of available API attributes](https://github.com/sapcc/go-bits/blob/53eeb20fde03c3d0a35e76cf9c9a06b63a415e6b/gopherpolicy/pkg.go#L151-L164)
 
 [os-env]: https://docs.openstack.org/python-openstackclient/latest/cli/man/openstack.html
-[os-pol]: https://docs.openstack.org/oslo.policy/latest/admin/policy-json-file.html
+[os-pol]: https://docs.openstack.org/oslo.policy/latest/admin/policy-json-file.html , https://docs.openstack.org/oslo.policy/latest/admin/policy-yaml-file.html
```

Comment:
That does not look like valid Markdown syntax. You can't have a link with two URLs at the same time.